templates/project/project-settings.html

```diff
@@ -0,0 +1,217 @@
+{% extends "project-base.html" %}
+{% load static %}
+{% load q %}
+
+{% block title %}
+{{ project.name }} - {{ title }}
+{% endblock %}
+
+{% block pagetitle %}
+<a href="{{project.get_absolute_url}}">{{ project }} ({{ acronym|safe }})</a>
+{% endblock %}
+
+{% block pageheader %}
+<h2 class="usa-prose-h2">Project Settings</h2>
+{% endblock %}
+
+{% block breadcrumbs_extension %}
+  <li class="usa-breadcrumb__list-item">
+    <a href="{{project.get_absolute_url}}" class="usa-breadcrumb__link">
+      <span>{{project.title}}</span>
+    </a>
+  </li>
+  <li class="usa-breadcrumb__list-item usa-current" aria-current="Project Settings">
+    <span>Project Settings</span>
+  </li>
+{% endblock %}
+
+{% block body_content %}
+
+  <section class="margin-bottom-4">
+    <p class="usa-intro">
```

Comment:
I know we talked about which uswds style to use for this text, but it seems like the usa-intro has a set font that deviates from the designs. I think we might need to custom size it instead.

Reply:
Ah, good catch!

Reply:
I added a font-family override to the cms.scss file. It is already in branding.